SDK/articles/CreatingAConnectedServiceExtension.md

```diff
@@ -167,9 +162,9 @@ The value of the attribute's AppliesTo parameter specifies what types of Visual
 
 You will implement the changes to the app developer's project in the AddServiceInstanceAsync method. We'll show you to do in a later section. For now, we simply return a simple instance of the AddServiceInstanceResult.
 
-F5 again, create a C# Web project, open the Connected Service dialog, and you will see the sample provider can now be configured.
+F5 again, create a C# Console App project, open the Connected Service dialog, and you will see the sample provider listed in the Connected Services tab
```

Comment:
This may need revisiting based on the outcome of the AppliesTo expression comment I left earlier.